HHS/Head-Start-TTADP
# [Prod] Create backend for QA dashboard, add tests for goal services, historical cleanup, new grant filter

## Description of change
- Cleanup historical training report data by removing extra detail from the JSON data field
- Correctly link objectives with their templates in historical data now that bug preventing this has been fixed
- Add new grant status filter (_Active, inactive, CDI_)
- Add some unit tests for goal services
- Remove unnecessary `src/.eslintrc` and `frontend/.eslintrc`.

## How to test
- For historical cleanup and test coverage, no testing is possible
- To test grant status filter, visit the regional dashboard, activity report page, and RTR search page and confirm that the filter is present and filters the correct data

## Issue(s)
* https://jira.acf.gov/browse/TTAHUB-2808
* https://jira.acf.gov/browse/TTAHUB-3361
* https://jira.acf.gov/browse/TTAHUB-2826
* https://jira.acf.gov/browse/TTAHUB-2977

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
